editor/connections_dialog.h

```diff
@@ -255,7 +255,7 @@ class ConnectionsDock : public VBoxContainer {
 	void _signal_menu_about_to_popup();
 	void _handle_slot_menu_option(int p_option);
 	void _slot_menu_about_to_popup();
-	void _rmb_pressed(const Ref<InputEvent> &p_event);
+	void _gui_input(const Ref<InputEvent> &p_event);
```

Comment:
```suggestion
	void _tree_gui_input(const Ref<InputEvent> &p_event);
```

I think it makes sense to emphasize that this is a callback of the `tree` node, not this node (otherwise we could override `gui_input`).

https://github.com/godotengine/godot/blob/26b1fd0d842fa3c2f090ead47e8ea7cd2d6515e1/scene/gui/control.h#L498